### PR TITLE
Replace Travis with GitHub Workflow

### PR DESCRIPTION
**Pull request by andrewmy**
CI runs here: https://github.com/andrewmy/stomp-php/pull/1

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        id: composer-cache
+        run: |
+          echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-
```
Comment (staabm) [inline .github/workflows/ci.yml:20]: caching could be simplified with https://github.com/ramsey/composer-install

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
```
Comment (staabm) [inline .github/workflows/ci.yml:50]: same here

File: .github/workflows/ci.yml (hunk 1 above)
Comment (andrewmy) [reply .github/workflows/ci.yml:20]: Good call, thanks!

**Comment by andrewmy** [comment]
`Stomp\Tests\Functional\ActiveMq\ClientTest::testHasFrameToRead` appears to be flaky :(

**Comment by andrewmy** [comment]
10s timeout helps 😁

**Comment by andrewmy** [comment]
Okay fixed it with a waiting script

**Comment by staabm** [comment]
would it make sense to PR a very small separate PR to get the Github Actions going, so we can see in this PR here if the proposed changes work?

**Comment by andrewmy** [comment]
@staabm I removed the extra fluff now, please check.
You can enable GH workflows for this repo to see output right here, or see https://github.com/andrewmy/stomp-php/pull/1/checks

**Comment by staabm** [comment]
Thx for your effort. Lets wait for an opinion of @jmglsn . Maybe he has an idea, why some tests are failling on GH but did not on travis ci

**Comment by jmglsn** [comment]
I'll have a look as well, sorry I just have like 5 minutes today :(

**Comment by andrewmy** [comment]
@jmglsn thanks for the merge, however there's still the issue with the flaky test :S
Also you need to enable GitHub Workflows for the repository to be able to use the CI again.

**Pull request merged**